pkg/pricer/pricer.go

```diff
@@ -189,6 +220,16 @@ func (s *Pricer) getPricingPeer(peer swarm.Address) (*pricingPeer, error) {
 	return peerData, nil
 }
 
+func (s *Pricer) storePriceTable(peer swarm.Address, priceTable []uint64) error {
+	s.logger.Debugf("Storing pricetable %v for peer %v", priceTable, peer)
```

Comment:
this is a nice-to-have, but if we choose to have it then please move it to `Trace`